boot/freeldr/freeldr/arch/i386/multiboot.S

```diff
@@ -173,23 +173,33 @@ mb3:
     mov ss, ax
 
     /* Jump to relocated code */
-    ljmp HEX(08), mb4
+    .byte HEX(EA)
+    .long mb4
+    .word HEX(08)
```

Comment:
What's the point of this change? It makes it less readable, because nobody knows what .byte HEX(EA) is.

Reply:
i reverted this